security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -276,6 +342,18 @@
         <help></help>
     </field>
     <field>
+      <label>DNS-01/Linode</label>
```

Comment:
Fabian is right. But FWIW this isn't going to be in the translation templates for a few months, hopefully by then we have a nicer solution.